### PR TITLE
Zelin/bms strobe light

### PR DESCRIPTION
**Pull request by zelinzhou**
added bms strobe light

### CODE REVIEW

File: src/ViewLayer/DebugDisplay/ControlPage/ControlUi/ControlUi.h [hunk 1]
```diff
@@ -38,6 +38,11 @@ class ControlUi : public I_ControlUi
     QLabel& resetOnLabel();
     QProgressBar& accelerationProgressBar();
     QProgressBar& regenBrakingProgressBar();
+    QLabel& bMSStrobeLightLabel();
```
[the remark this answers is not in the document]
Comment (chordatgh1234) [reply src/ViewLayer/DebugDisplay/ControlPage/ControlUi/ControlUi.h]: I think the label name in the ui file needs to be changed too.

File: src/ViewLayer/DebugDisplay/ControlPage/ControlView/ControlView.cpp [hunk 2]
```diff
@@ -93,7 +93,8 @@ void ControlView::connectDriverControls(DriverControlsPresenter& driverControlsP
             this, SLOT(reverseReceived(bool)));
 
     connect(&driverControlsPresenter, SIGNAL(resetReceived(bool)),
-            this, SLOT(resetReceived(bool)));
+            this, SLOT(rese
```
Comment (bill-luu) [inline src/ViewLayer/DebugDisplay/ControlPage/ControlView/ControlView.cpp]: There was a new line accidentally inserted here.

**Comment by bill-luu** [comment]
Travis is failing again

**Comment by bill-luu** [comment]
In the future, please look into adding more descriptive commit messages and potentially rebasing to change past messages. Later on, going through the code will be much better if the logs are more descriptive and easy to read. 

[Here is a good guide on commit conventions.](https://chris.beams.io/posts/git-commit/)

**Comment by chordatgh1234** [comment]
Travis is showing a astyle error, try running the format.sh script in the Episilon-Dashboard folder.

**Comment by zelinzhou** [comment]
Yes. I did run the format.sh script in the Epision-Dashboradfolder in
terminal. Should I push it then?

Justin Hung <notifications@github.com> 于2019年2月2日周六 下午12:08写道：

**Pull request merged**